### PR TITLE
link to GH issue. Add 400 Bad Request

### PR DESCRIPTION
**Pull request by kkraune**
https://github.com/vespa-engine/vespa/issues/13465

### CODE REVIEW

File: documentation/reference/document-v1-api-reference.html [hunk 1]
```diff
@@ -273,18 +273,27 @@ <h2 id="request-body">Request body</h2>
 
 
 <h2 id="http-status-codes">HTTP status codes</h2>
-Non-exhaustive list of status codes:
+<p>
+  See <a href="https://github.com/vespa-engine/vespa/issues/13465">13465</a>
+  for defined document types not assigned to a content cluster.
```
Comment (jonmv) [inline documentation/reference/document-v1-api-reference.html:278]: I think this is too specific to put here. _Perhaps_ something to list under the description of code 400.

File: documentation/reference/document-v1-api-reference.html [hunk 2]
```diff
@@ -273,18 +273,27 @@ <h2 id="request-body">Request body</h2>
...
+</p><p>
+  Non-exhaustive list of status codes:
+</p>
 <table class="table">
 <tr>
   <th>200</th>
   <td>OK. Note that updating a non-existing document is defined as success.</td>
+</tr><tr>
+  <th>400</th>
+  <td>Bad request. Returned for undefined document types.
```
Comment (jonmv) [inline documentation/reference/document-v1-api-reference.html:288]: It's returned to various input errors, not just document types without a cluster.

File: documentation/reference/document-v1-api-reference.html (hunk 1 above)
Comment (kkraune) [reply documentation/reference/document-v1-api-reference.html:278]: agree, move there for now

File: documentation/reference/document-v1-api-reference.html (hunk 2 above)
Comment (kkraune) [reply documentation/reference/document-v1-api-reference.html:288]: reformulated

**Review by jonmv — APPROVED** [review]

**Pull request merged**